src/__tests__/perseus-markdown-test.jsx

```diff
@@ -63,8 +63,10 @@ var htmlFromMarkdown = function(source) {
 var assertParsesToReact = function(source, html) {
     var actualHtml = htmlFromMarkdown(source);
     if (actualHtml !== html) {
+        /* eslint-disable no-console */
```

Comment:
Somewhere we have a separate configuration for things that are run in a `node` environment, where `console.*` are fine. We should really be using that for things in `__tests__` instead of whitelisting these. But that's for another day :)